WordPressKit.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name          = "WordPressKit"
-  s.version       = "4.34.0"
+  s.version       = "4.34.1"
```

Comment:
Should this be 4.34.1-beta.1?

Reply:
Actually no. Since I'll be targeting the `release` branch on WPiOS this should be a hotfix. :)

Reply:
Gotcha. Thanks for the clarification!